tasks/setup-dnf.yml

```diff
@@ -6,12 +6,33 @@
                           else cockpit_packages }}"
   when: cockpit_packages is defined
 
-- name: Ensure Cockpit Web Console packages are installed.
+- name: Get cockpit-bridge available version in "full" package install mode
+  when: cockpit_packages == "full"
+  changed_when: false
+  command: dnf repoquery --latest-limit=1 --queryformat '%{version}' cockpit-bridge
+  register: __cockpit_bridge_version
+
+- name: Initialize dynamic excludes
+  set_fact:
+    cockpit_packages_dyn_exclude: []
+
+# cockpit-pcp was merged into cockpit-bridge in version 326, and thus
+# and conflicts with current bridge; but CentOS 9 composes have older versions
+# published, this leads to installation conflicts
+# https://issues.redhat.com/browse/CS-2799
+- name: Ignore cockpit-pcp in "full" package install mode with Cockpit ≥ 326
+  when:
+  - cockpit_packages == "full"
```

Comment:
```suggestion
    - cockpit_packages == "full"
```

Reply:
Ah, thanks for spotting! Obsoleted with Rich's rewrite below, this went away. But good to know for next time.